noWord/plugins/ForeachBlock/ForeachBlock.py

```diff
@@ -84,9 +84,10 @@ def makeForeach(self, context, path, resource, keys, name, subblocks, keepConten
             subcontent.extend(context.processFuncObj(subblocks, context, path))
 
             if keepContentTogether:
-                content.extend([KeepTogether(subcontent)])
-            else:
-                content.extend(subcontent)
+                for index in range(0, len(subcontent)-2):
+                    subcontent[index].keepWithNext = True
```

Comment:
I couldn't find this property in reportlab's documentation. Can you provide some insight on this mechanism ?

Reply:
Based on source code analysis and this post:
https://stackoverflow.com/questions/4797536/how-to-group-objects-in-reportlab-so-that-they-stay-together-across-new-pages

Probably an undocumented feature, but the KeepTogether flowable is really not working...